docs/docs/refs-and-the-dom.md

```diff
@@ -73,7 +73,23 @@ class AutoFocusTextInput extends React.Component {
 }
 ```
 
-You may not use the `ref` attribute on functional components because they don't have instances. You can, however, use the `ref` attribute inside the functional component:
+**You may not use the `ref` attribute on functional components** because they don't have instances. In the below code the ref will always be called with `null`:
+
+```
+function StatelessComponent() {
+  return <div>hello</div>
+}
+
+class ParentComponent extends React.Component {
+  render() {
+    return <StatelessComponent ref={node => this.helloNode = node} />
```

Comment:
Good to add an inline comment here:

```js
// This will not work!
```

Otherwise people will just read over this and remember this example.